### DISCUSSION TITLE
Rodar como serviço, ou vcl minimizado?

### DISCUSSION THREAD

**Discussion by linces** [HashLoad/horse#153] [Q&A] [answered]
Bom dia pessoal,

É possível rodar o servidor horse como um serviço? Ou em VCL? Só vejo exemplos console app.

**Comment by CarlosHe** [comment, marked as answer]
Olá! Claro, é possível sim!

**Comment by CarlosHe** [comment]
Veja na pasta de samples do nosso repositório.

**Reply by rodrigoferreiramonteiro** [reply]
Ola, tenho testado o exemplo em lcl do lazarus e a aplicação para de responder depois do listem 

![Screenshot_11](https://user-images.githubusercontent.com/4587027/187806887-c127d3b8-30f1-4822-9b0e-a1b9c37ae15f.png)

**Reply by viniciussanchez** [reply]
Isso já foi ajustado pelo Alexandre Magno, mas ainda não subimos a alteração. Em breve esse problema de congelamento estará resolvido

**Reply by RobertoSobreira01** [reply]
> Isso já foi ajustado pelo Alexandre Magno, mas ainda não subimos a alteração. Em breve esse problema de congelamento estará resolvido

no ambiente lazarus v3.4 - x86_64-linux-gtk2, FPC Version 3.2.2  - horse 3.1.6   está travando neste ponto..  alguma dica

**Reply by RobertoSobreira01** [reply]
> Isso já foi ajustado pelo Alexandre Magno, mas ainda não subimos a alteração. Em breve esse problema de congelamento estará resolvido

tem que colocar -dHORSE_LCL   
![Captura de tela 2024-07-11 113812](https://github.com/HashLoad/horse/assets/117688318/d188262f-bb2d-42ce-aab3-0150bba460ac)

**Comment by linces** [comment]
Fantástico, eu fui fazendo tudo utilizando o boss, nem me liguei em vir aqui e procurar os exemplos.. 

Valeu.
